trivy/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.12
+
+# Install curl,kubectl
+RUN apk add curl && \
+   curl -o /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+   chmod +x /bin/kubectl
+
+# Install trivy
+RUN curl -sfL https://raw.githubusercontent.com/kyeswanth/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
```

Comment:
Can we put this script in this repository and map it in locally?

Reply:
Updated PR with latest changes